src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Options/CollectionRuleOptionsExtensions.cs

```diff
@@ -168,6 +168,30 @@ public static CollectionRuleOptions AddLoadProfilerAction(this CollectionRuleOpt
                 });
         }
 
+        public static CollectionRuleOptions AddSetEnvironmentVariableAction(this CollectionRuleOptions options, Action<SetEnvironmentVariableOptions> configureOptions)
+        {
+            return options.AddAction(
+                 KnownCollectionRuleActions.SetEnvironmentVariable,
+                 callback: actionOptions =>
+                 {
+                     SetEnvironmentVariableOptions setEnvOpts = new();
+                     configureOptions?.Invoke(setEnvOpts);
+                     actionOptions.Settings = setEnvOpts;
+                 });
+        }
+
+        public static CollectionRuleOptions AddGetEnvironmentVariableAction(this CollectionRuleOptions options, Action<GetEnvironmentVariableOptions> configureOptions)
```

Comment:
Rather than using a fluent-style API here, just have `string name` parameter and set it within the method.